README.md

```diff
@@ -1,17 +1,23 @@
 # acr-base-importer
 Pipelines to automatically import updates to base images and scan them for vulnerabilities
 
-# ACR Cache
-The pipeline will also create ACR Caches into hmctspublic registry.
+# ACR Cache Rules
```

Comment:
this should be after upstream image, user documentation before developer documentation (and a lower heading level)